sdks/typescript/featurehub-javascript-client-sdk/app/edge_featurehub_config.ts

```diff
@@ -67,11 +67,11 @@ export class EdgeFeatureHubConfig implements FeatureHubConfig {
   }
 
   public clientEvaluated(): boolean {
-    return this._apiKey.includes('*');
+    return this._clientEval;
   }
 
   getApiKeys(): string[] {
-    return Object.assign([], this._apiKeys);
+    return this._apiKeys;
```

Comment:
no, this is wrong, the original is correct, it creates a copy of the apikeys. This one makes the internal array available.